http/router_utils.go

```diff
@@ -247,3 +249,41 @@ func handleClusterNameError(writer http.ResponseWriter, err error) {
 func SplitRequestParamArray(arrayParam string) []string {
 	return strings.Split(arrayParam, ",")
 }
+
+// ReadClusterListFromPath retrieves list of clusters from request's path
+// if it's not possible, it writes http error to the writer and returns false
+func ReadClusterListFromPath(writer http.ResponseWriter, request *http.Request) ([]string, bool) {
+	rawClusterList, err := GetRouterParam(request, "cluster_list")
```

Comment:
According to https://github.com/RedHatInsights/insights-results-smart-proxy/blob/ebf45c0dba7e31516b1fd7d3696fd28d91e6562c/openapi.json#L366, it should be `clusterList`

Reply:
It's a nitpick, probably it is better to change the openapi.json spec rather than this

Reply:
yup, will do